mlflow/pytorch/__init__.py

```diff
@@ -591,6 +696,34 @@ def _load_model(path, **kwargs):
             return torch.jit.load(model_path)
 
 
+@experimental
+def load_state_dict(model_uri):
+    """
+    Load a PyTorch model state dict from a local file or a run.
```

Comment:
Can we simply say `Load a state dict ...` because the state dict might not be a model?